File: 1198.py
while True:
    try:
        y=input().split()
        y[0]=int(y[0]);y[1]=int(y[1])
        print(y[0]-y[1]) if y[0]-y[1]>0 else print((y[0]-y[1])*-1)
    except:break

File: 5-Matemcatica/1198-O_Bravo_Guerreiro_Hashmat-5/1198.cs
using System;

public class HelloWorld
{
    public static void Main(string[] args)
    {
        while(true){
            try{
                string[] nums = Console.ReadLine().Split(' ');
                long result= long.Parse(nums[1]) - long.Parse(nums[0]);
                if (result < 0){result*=-1;}
                Console.WriteLine(result);
            }
            catch(Exception ex){
                break;
            }
        }
    }
}